packages/cli/src/tools/installPods.js

```diff
@@ -17,6 +17,10 @@ async function installPods({
   loader?: typeof Ora,
 }) {
   try {
+    if (!(await fs.pathExists('ios'))) {
```

Comment:
You can use `https://nodejs.org/api/fs.html#fs_fs_existssync_path`. The `pathExistsSync` version from `fs-extra` is just its alias.

Ref: https://github.com/jprichardson/node-fs-extra/blob/master/docs/pathExists-sync.md